packages/jaeger-ui/src/components/TracePage/TraceSpanView/index.tsx

```diff
@@ -133,7 +133,7 @@ export default class TraceSpanView extends Component<Props, State> {
     const columns: ColumnProps<Span>[] = [
       {
         title: 'Service Name',
-        dataIndex: 'process.serviceName',
+        dataIndex: ['process', 'serviceName'],
```

Comment:
is this our own notation or something that antd component supports, like a nested indexing path into a data structure?

Reply:
antd `dataIndex` definition changes from `'xxx.yyy' to ['xxx', 'yyy']`.
So basically, antd now want us to supply dataIndex path in the form of an array for nested indexing pathnames.

Here's a link to that: https://4x.ant.design/docs/react/migration-v4#:~:text=Nest%20dataIndex%20definition%20changes%20from%20%27xxx.yyy%27%20to%20%5B%27xxx%27%2C%20%27yyy%27%5D